improver_tests/calibration/dz_rescaling/test_estimate_dz_rescaling.py

```diff
@@ -246,6 +246,9 @@ def test_estimate_dz_rescaling(
     truth_adjustment is applied to the truth data to show how larger and inverted
     data (w.r.t. height) behave.
     """
+    if n_frts > 1 and len(forecast_periods) > 1:
+        return
+
```

Comment:
Is there a catch for this in operational use, or would it simply not happen?

Reply:
We don't process cubes of this shape in practice. As shown in the acceptance test data, the current set-up involves possibly multiple forecast reference times for a single lead time. We never process multiple forecast reference times and multiple forecast periods at once.